cmd/server/main.go

```diff
@@ -154,7 +157,26 @@ func buildCLI() *cli.App {
 					services = strings.Split(c.String("services"), ",")
 				}
 
-				cfg, err := config.LoadConfig(env, configDir, zone)
+				var cfg *config.Config
+				var err error
+
+				switch {
+				case c.IsSet("config-file"):
+					configFilePath := c.String("config-file")
+					if !filepath.IsAbs(configFilePath) {
+						configFilePath = filepath.Join(c.String("root"), configFilePath)
```

Comment:
Let's also deprecate `root`, I don't think we need it for anything but resolving the config path. Load the config using the OS's default path resolution logic.